geode-core/src/main/java/org/apache/geode/internal/cache/LeafRegionEntry.cpp

```diff
@@ -239,9 +239,6 @@ public class LEAF_CLASS extends PARENT_CLASS {
   @Override
   public void returnToPool() {
     // Deadcoded for now; never was working
```

Comment:
There is now no dead code in this method.

Reply:
I'll change it to `// never implemented` but ultimately we should probably remove this method.